### ISSUE TITLE
lint shows "redeclare" error when using the same name in another file

### ISSUE THREAD

**Issue by kyeotic** [denoland/deno_lint#1043]
### Lint Name
`Cannot redeclare block-scoped variable '_a'.deno-ts(2451)`

### Code Snippet

Place this in two files.

```ts
const _a = '1'
```

### Expected Result
No error

### Actual Result
`Cannot redeclare block-scoped variable '_a'.deno-ts(2451)`

### Additional Info
This is happening in VS Code with the latest Deno extension.
Any two files sharing a variable produce the error. They don't even need to be in the same folder, just open at the same time.

![Kapture 2022-05-11 at 17 19 57](https://user-images.githubusercontent.com/1187432/167967909-475d90dd-3049-4c3e-bf82-302f4720b996.gif)

I understand that this might be related to [this issue](https://github.com/microsoft/TypeScript/issues/18232), but since deno only supports modules this issue should not apply. All files should be in an isolated scope when using deno_lint because thats how they will be handled by deno.

### Version

```
❯ deno --version
deno 1.21.2 (release, aarch64-apple-darwin)
v8 10.0.139.17
typescript 4.6.2
```

**Comment by bartlomieju** [comment]
CC @kitsonk

**Comment by magurotuna** [comment]
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/23649474/194753130-32dcb22f-499c-4c24-8fd4-42a4ca5d50ac.png">
 
Confirmed this issue was resolved already at some point, so closing now. Let us know if there's still an issue.

extension v3.13.1
deno 1.26.1 (release, aarch64-apple-darwin)
v8 10.7.193.3
typescript 4.8.3
